### ISSUE TITLE
Idea: graphical output

### ISSUE THREAD

**Issue by ionelmc** [ionelmc/python-hunter#71]
Use graphviz or something to get call graphs instead of text output.

**Comment by wsdookadr** [comment]
> Use graphviz or something to get call graphs instead of text output.

Yes, this would be a really nice feature. [I wrote a custom Action here](https://stackoverflow.com/a/65147704/827519) that's capable of building a call graph, and then wrote some code that renders it via [graphviz](https://graphviz.org/), it's a bit custom for that specific use-case but I think it could be made more generic. Maybe it can help as a prototype.

I've worked with large call graphs in other contexts, and I've seen they can get quite big. I think in addition to graphviz, it would be helpful to produce the graphs in other formats too, and to render them with [gephi](https://gephi.org/), [holoviews](https://holoviews.org/) . I think I'm still looking for something that allows the navigation of a graph without bringing all of it into the viewport (maybe holoviews does that, I haven't checked yet).
